reference/matrix/sellp_kernels.cpp

```diff
@@ -215,8 +219,10 @@ void fill_in_dense(std::shared_ptr<const ReferenceExecutor> exec,
             }
             for (size_type i = slice_sets[slice]; i < slice_sets[slice + 1];
                  i++) {
-                result->at(global_row, col_idxs[row + i * slice_size]) +=
-                    vals[row + i * slice_size];
+                const auto col = col_idxs[row + i * slice_size];
+                if (col != invalid_index<IndexType>()) {
+                    result->at(global_row, col) += vals[row + i * slice_size];
```

Comment:
I guess if we don't allow ell to have multiple entries with the same col idx, we shouldn't for sellp as well?
```suggestion
                    result->at(global_row, col) = vals[row + i * slice_size];
```